docs/units-and-global-variables.rst

```diff
@@ -135,6 +135,7 @@ ABI Encoding and Decoding Functions
 - ``abi.encode(...) returns (bytes memory)``: ABI-encodes the given arguments
 - ``abi.encodePacked(...) returns (bytes memory)``: Performs :ref:`packed encoding <abi_packed_mode>` of the given arguments. Note that packed encoding can be ambiguous!
 - ``abi.encodeWithSelector(bytes4 selector, ...) returns (bytes memory)``: ABI-encodes the given arguments starting from the second and prepends the given four-byte selector
+- ``abi.encodeCall(function functionPointer, (...)) returns (bytes memory)``: ABI-encodes a call to ``functionPointer`` with the arguments found in the tuple. Performs a full type-check, ensuring the types match the function signature.
```

Comment:
Maybe you could add ```equivalent to `bytes.concat(functionPointer.selector, abi.encode(...))` ```?

Reply:
Also there is another file called `cheatsheet.rst` that lists these functions as well.

Reply:
Sorry, it's of course `encodeWithSelector`